scipy/special/basic.py

```diff
@@ -754,20 +754,97 @@ def riccati_yn(n, x):
 
 
 def erfinv(y):
-    """Inverse function for erf.
+    """Inverse error function.
+    This function compute the inverse error function.
+    Inverse error function is satisfying :math:`erf(erf^{-1}(x))=x`.
+
+    Parameters
+    ----------
+    y : ndarray
+        Argument at which to evaluate for (-1, 1)
+
+    Returns
+    -------
+    erfinv : ndarray
+        Value of erfinv(y0), ..., erfinv(yn)
```

Comment:
The array like object `y` may have many dimensions, not just be a single array, and the indices would run from `0` to `n-1`. You could just write this as
`The inverse erf of y, element-wise`.